src/functional/histogram_compression.zig

```diff
@@ -566,9 +787,9 @@ test "PWCH can compress and decompress" {
     const allocator = testing.allocator;
     var uncompressed_values = ArrayList(f64).init(allocator);
     defer uncompressed_values.deinit();
-    try tester.generateBoundedRandomValues(&uncompressed_values, 0.0, 1.0, random);
+    try tester.generateBoundedRandomValues(&uncompressed_values, 0.0, 10.0, random);
 
-    const error_bound: f32 = @floatFromInt(uncompressed_values.items.len);
+    const error_bound: f32 = 10;
```

Comment:
Well, there are only two times where the error bound is defined, and since the number 10 is bound to the way the points values are generated I think is better to leave this way.